src/current/v25.3/temporary-tables.md

```diff
@@ -40,6 +46,10 @@ Because the [`SHOW TABLES`]({% link {{ page.version.version }}/show-tables.md %}
 
 ## Examples
 
+{{site.data.alerts.callout_info}}
+For intermediate results, consider using [common table expressions (CTEs)]({% link {{ page.version.version }}/common-table-expressions.md %}#overview) instead of temp tables. For more information, see [Performance considerations](#performance-considerations).
```

Comment:
```suggestion
For intermediate results, consider using [common table expressions (CTEs)]({% link {{ page.version.version }}/common-table-expressions.md %}#overview) instead of temporary tables. For more information, see [Performance considerations](#performance-considerations).
```

Reply:
same as above re: leaving since defined in page overview